docs/code-signing.rst

```diff
@@ -0,0 +1,10 @@
+Code signing policy
+===================
+
+The code used by the DebOps Project and commited in its repositories is
+expected to be signed by a valid GPG key of a DebOps Developer or DebOps
+Contributor. This should allow for secure code authentication and a valid audit
```

Comment:
On the other hand, creating a PGP key is not difficult. It should also be doable for contributes at least to a later time. The difficult part with PGP is to keep your private key private :wink: